src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -165,63 +169,63 @@
         GROUP BY s.cancer_study_identifier, sl.stable_id, sl.name;
     </select>
 
-
-    <sql id="getClinicalDataCountsQuerySample">
+    <sql id="getClinicalDataCountsQuery">
+        (
+        WITH clinical_data_query AS (
         SELECT
-        attribute_name as attributeId,
-        <include refid="normalizeAttributeValue">
-            <property name="attribute_value" value="attribute_value"/>
-        </include> as value,
```

Comment:
Don't we want to normalize NA values (like N/A, NAN, etc.) anymore? Did we completely remove them from the database?

Reply:
Yea, weird how it shows it removed. I have it here in line 181

`            AND <!-- Table creation in clickhouse.sql has ensured no NA values but extra caution is always appreciated -->
            <include refid="normalizeAttributeValue">
                <property name="attribute_value" value="value"/>
            </include>`